src/sql/workbench/contrib/dashboard/browser/services/breadcrumb.service.ts

```diff
@@ -61,8 +61,10 @@ export class BreadcrumbService implements IBreadcrumbService {
 	}
 
 	private getDbBreadcrumb(profile: ConnectionProfile): MenuItem {
+		let defaultDatabaseName = profile.providerName === 'LOGANALYTICS' ? 'workspace-name' : 'database-name';
```

Comment:
When is this actually showing up? Doesn't seem like something we should generally be displaying I'd think

Reply:
It gets displayed in the Dashboard when there's no database on a cluster. For Azure Monitor, there is never a database because we're connecting a Workspace directly and a workspace is effectively a database.